### PR TITLE
feat: add presentation tools and database

### PR DESCRIPTION
**Pull request by julianguyn**

### CODE REVIEW

File: docs/General/Presentations/index.md [hunk 1]
```diff
@@ -1,7 +1,42 @@
...
+**Key Slides**
+
+- **Title slide**: Include your name, date, affiliation, event/conference, and contact information (e.g. email). Optional: include the BHK Lab logo and others (e.g. UHN, University of Toronto, etc)
+- **Acknowledgements slide**: Include names of lab members (or can reference 'BHK Lab') and other collaborators along with your institutional affilications/funding sources (e.g. UHN, CIHR, NSERC)
+
+**Other slides**
+
+- All slides should include slide numbers
+- Slide headings should be short (1-2 lines max) and descriptive
+- Use visuals whenever possible (feel free to take from existing presentations *Link database page)
```
Comment (mtran-code) [inline docs/General/Presentations/index.md:14]:
```suggestion
- Use visuals whenever possible (feel free to take from existing presentations in the [database](database))
```

File: docs/General/Presentations/index.md [hunk 2]
```diff
@@ -1,7 +1,42 @@
...
+
 
 ## Poster Standards
 
-## Talk Standards
+**Components**
+
+- **Heading**: Include title, author(s) name, supervisor name (if student poster) across the top border
+- **Abstract** (optional): Brief paragraph (~250 words) outlining purpose, methods, results, and conclusions
+- **Introduction**: Background information pertinent to your project to help audience understand motivation. Outline the goal, objective(s), and hypotheseis(es) of your research.
```
Comment (mtran-code) [inline docs/General/Presentations/index.md:23]: Typo
```suggestion
- **Introduction**: Background information pertinent to your project to help audience understand motivation. Outline the goal, objective(s), and hypothesis(es) of your research.
```

File: docs/General/Presentations/index.md [hunk 3]
```diff
@@ -1,7 +1,42 @@
...
+- **Materials and Methods**: Brief outline of materials and methods used in your work, listed clearly and logically.
+- **Results**: Present data in photographic, graphical, or tabular form. Include descriptive figure titles and avoid lengthy captions. Be sure to not include abbreviations not explained in the text. 
+- **Discussion**: Address results and describe relevance to objective(s) and hypothesis(es).
+- **Conclusions**: Stated clearly and concisely, addressing project objectives and stating overall significance. 
+- **References**: All references (publications)
+- **Acknowledgements**: Any key lab memebers, collaborators, and funding sources.
```
Comment (mtran-code) [inline docs/General/Presentations/index.md:29]: Typo
```suggestion
- **Acknowledgements**: Any key lab members, collaborators, and funding sources.
```

File: docs/General/Presentations/index.md [hunk 4]
```diff
@@ -1,7 +1,42 @@
 # Presentation Standards
 
-TODO: #35 Add Presentation Standards
+## Talk Standards
+
+**Key Slides**
```
Comment (mtran-code) [inline docs/General/Presentations/index.md:5]: Might be nicer to use markdown subheadings instead of bold text? This would apply to all the subheadings on the page.

```suggestion
### Key Slides
```

File: docs/General/Presentations/index.md [hunk 5]
```diff
@@ -1,7 +1,42 @@
...
+Size of lettering must be large enough to be legible from approximately 2m. Use a clear and simple font between 18 point and 30 point in size. We recommend the following for Arial:
+
+- 18 point: best viewed at 1m (for figure titles, legends, acknowledgements, etc)
+- 24 point: best viewed at 2m (for main text)
+- 30 point: best viewed at 3m (for section headings)
+
+**Suggestions**
+
+- Avoid overloading with text, use bullet points where possible
+- Whenever appropriate, use clear diagrams, figures, and table (e.g. materials and methods)
```
Comment (mtran-code) [inline docs/General/Presentations/index.md:42]: Grammar consistency
```suggestion
- Whenever appropriate, use clear diagrams, figures, and tables (e.g. materials and methods)
```

File: docs/General/Presentations/tools.md [hunk 6]
```diff
@@ -1,10 +1,38 @@
...
+
+
+**Miro**
+
+[Miro](https://miro.com/login/) is a collaborative white board platform that enables simple diagram and multi-figure panel creation for presentations and/or manuscripts. For diagrams that will be used and updated by others in the lab, please request a Miro Board be created via the BHK Lab Miro account.
+
+
+**Images and Icons**
+
+The following are open-source biological imaging databases and platforms. The BHK Lab holds a premium Flaticon account (*do I provide login details?*). If Biorender and/or BioArt are used, please remember to add them to the acknowledgements.
```
Comment (mtran-code) [inline docs/General/Presentations/tools.md:28]: I think the site is technically public so probably don't include login details. I would put something like "Contact Sisira/whoever for login details", etc.

File: docs/General/Presentations/database.md
Comment (mtran-code) [inline docs/General/Presentations/database.md:1]: There's probably a better way to format the slide information, but that can be the scope of another PR. Otherwise looks good!

Comment (julianguyn) [reply docs/General/Presentations/database.md:1]: Thank you! Wasn't sure what the best way was - I need to discuss with Ben
